host/src/lib/HelperFunctions.jsx

```diff
@@ -130,7 +139,7 @@ export const getMultiChoiceAnswers = (
     const answers = extractAnswers(teamsArray, currentState, currentQuestionId);
     answers.forEach(({ team, answer }) => {
       choices.forEach((choice) => {
-        if (answer.answerContent.rawAnswer === choice.text) {
+        if (answer.answer.rawAnswer === choice.text) {
           const index = confidenceLevelsArray.indexOf(
```

Comment:
This is typical throughout the code. We're using `answer` instead of `answerContent` to reference `NumericAnswer`, `StringAnswer` etc in the `BaseAnswerClass`.